DuckDuckGo/Autoconsent/userscript-exp.js

```diff
@@ -0,0 +1,19 @@
+import AutoConsent from '@duckduckgo/autoconsent/extra';
```

Comment:
This file is just a copy of `DuckDuckGo/Autoconsent/userscript.js` with one change - in line 1 we use `'@duckduckgo/autoconsent/extra'` instead of `'@duckduckgo/autoconsent'`, which generates the larger bundle including easylist filters.